### DISCUSSION TITLE
2022/09/26/2022-sept-cf-tutorials-4/

### DISCUSSION THREAD

**Discussion by giscus** [DeCalvin2006/blog#7] [General]
# 2022/09/26/2022-sept-cf-tutorials-4/

这是 Codeforces 题解系列的第 ④ 篇。

https://decalvin.netlify.app/2022/09/26/2022-sept-cf-tutorials-4/

**Comment by DeCalvin2006** [comment]
终于全写完了！！！
后续大概也会继续更新的QwQ
